lib/s3routes/routesUtils.ts

```diff
@@ -195,12 +193,10 @@ export const JSONResponseBackend = {
         additionalHeaders?: { [key: string]: string } | null,
     ) {
         const bytesSent = Buffer.byteLength(json);
-        log.trace('sending success json response');
         log.addDefaultFields({ bytesSent });
         setCommonResponseHeaders(additionalHeaders, response, log);
         response.writeHead(200, { 'Content-type': 'application/json' });
-        log.debug('response http code', { httpCode: 200 });
-        log.trace('json response', { json });
+        log.trace('sending success json response', { json });
```

Comment:
* should we not keep the `httpCode` field? (may be consistent with other error logs)
* my understanding of the "duplicated" traces is that we want(ed) a debug log in anycase, but include JSON only when log level is trace : should we try to keep such behavior (i.e. changing the behavior based on `logger.config.level()`), or do we really _want_ to remove the debug log?

Reply:
We have the debug log when the response is complete just below, with the real status code (while the raw 200 is incorrect some times, could be 2xx). Meaning that, in case of debug log level, we end up with both the 200 and the real code sent to the client, which is not useful. This is removed on purpose, this brings 0 benefit as a log.

Then we keep a trace as before with the json info.

If we have an error log we should not enter the okResponse anyway (or it's a multi delete).